articles/api-auth/dynamic-client-registration.md

```diff
@@ -0,0 +1,236 @@
+---
+description: How to dynamically register Clients with Auth0, using the Management APIv2.
+toc: true
+---
+
+# Dynamic Client Registration
+
+Dynamic Client Registration is a feature, based on the [OpenID Connect Dynamic Client Registration specification](https://openid.net/specs/openid-connect-registration-1_0.html), that enables you to register clients dynamically. These clients are called **third party clients** (since they are registered and used by third party developers).
+
+## Third Party Client characteristics
+
+Third party clients have the following characteristics:
+
+- The [ID tokens](/tokens/id-token) generated for these clients, hold minimum user profile information.
+
+- They can use only tenant level connections (domain connections). These are sources of users, configured in the tenant's [dashboard](${manage_url}) as connections. These connections are enabled for every third party client and can be also enabled for selected first party (standard) clients.
+
+- To authenticate users using [Lock](/libraries/lock), you will have to use a version greater than `10.7`.
+
+- They cannot skip user consent when consuming APIs. This is for security purposes, as anyone can create a client, but each client relays on the final user to provide consent.
+
+- They cannot use [ID tokens](/tokens/id-token) to invoke [Management APIv2](/api/management/v2) endpoints. Instead, they should get a Management APIv2 Token (see the *How to get a Management APIv2 Token* panel for details). Note that the client should be granted the `current_user_*` scopes, as required by each endpoint.
+  - `read:current_user`: [List or search users](/api/management/v2#!/Users/get_users), [Get a user](/api/management/v2#!/Users/get_users_by_id), [Get user Guardian enrollments](/api/management/v2#!/Users/get_enrollments)
+  - `update:current_user_metadata`: [Update a user](/api/management/v2#!/Users/patch_users_by_id), [Delete a user's multifactor provider](/api/management/v2#!/Users/delete_multifactor_by_provider)
+  - `delete:current_user_metadata`: [Delete a user](/api/management/v2#!/Users/delete_users_by_id)
```

Comment:
Remove `delete:current_user_metadata` and `create:current_user_metadata` from the list. While auth0-server will return these, they don't have a proper use case in API2.